ros2pkg/ros2pkg/api/create.py

```diff
@@ -0,0 +1,94 @@
+# Copyright 2017 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from io import StringIO
+import os
+import sys
+
+import em
+
+
+RELATIVE_RESOURCE_PATH = '../../resource/'
+
+
+def expand_template(template_file, data, output_file, minimum_timestamp=None):
```

Comment:
I don't see why this use case would need the `minimum_timestamp` logic. Since it doesn't seem to be used it should be removed.